modules/datastore/src/Storage/DatabaseTable.php

```diff
@@ -181,15 +181,15 @@ private function buildTableSchema($tableName, $fieldsInfo) {
    *
    * @param string $type
    *   Type returned from the describe query.
-   * @param string|null $extra
-   *   MySQL "Extra" property for column.
+   * @param mixed $extra
+   *   Additional information for column.
    *
    * @return string
    *   Fritionless Table Schema compatible type.
    *
    * @see https://specs.frictionlessdata.io/table-schema
```

Comment:
Slight pre-existing error in DocBlock return type and description.
```suggestion
   * @return array
   *   Drupal Schema array.
   *
   * @see https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Database!database.api.php/group/schemaapi/9.2.x
```